pipeline/trusted-resources/pkg/trustedtask/trustedtaskrun.go

```diff
@@ -64,8 +55,12 @@ var (
 	_ apis.Validatable = (*TrustedTaskRun)(nil)
 )
 
-// Validate the TaskRun is tampered or not.
+// Validate the Taskref referred task is tampered or not. Only support OCI task bundle now.
 func (tr *TrustedTaskRun) Validate(ctx context.Context) (errs *apis.FieldError) {
+	if !apis.IsInCreate(ctx) {
+		return nil
+	}
```

Comment:
I'd break this change into another PR - this is a different change than just signing Tasks, and we should make sure to add tests to validate this behavior

Reply:
np!